openstack/cronus/templates/maillog/deployment.yaml

```diff
@@ -0,0 +1,76 @@
+{{- if .Values.maillog.enabled }}
+{{- if .Values.maillog.deployment.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: maillog
+  {{- if .Values.maillog.annotations }}
+  annotations:
+    {{ toYaml .Values.maillog.annotations | indent 4 }}
+  {{- end  }}
+  {{- if .Values.maillog.labels }}
+  labels:
+    {{ toYaml .Values.maillog.labels | indent 4 }}
+  {{- end }}
+spec:
+  replicas: {{ default .Values.maillog.deployment.replicas }}
+  strategy:
+{{- if eq "RollingUpdate" .Values.maillog.updateStrategy }}
+    rollingUpdate:
+      maxSurge: {{ .Values.maillog.maxSurge }}
+      maxUnavailable: {{ .Values.maillog.maxUnavailable }}
+{{- end }}
+    type: {{ .Values.maillog.updateStrategy }}
+  selector:
+    matchLabels:
+      name: maillog
```

Comment:
maybe best that the match labels line up through a variable, this way there is not chance for typos. I.E. using `{{.Chart.Name}}` to match. And just curios: why have the label `instance: maillog`?